test/passing/ocp_indent_compat.ml

```diff
@@ -2,3 +2,10 @@
 
 (* Bad: unboxing the function type *)
 external i : (int -> float[@unboxed]) = "i" "i_nat"
+
+module type M = sig
+  val doc_atrs :
+        (string Location.loc * payload) list
```

Comment:
I suspect this does not work when piped to ocp-indent

Reply:
I'm working on writing the colon on the line below